.github/actions/build-infinispan/action.yml

```diff
@@ -0,0 +1,37 @@
+name: Build Infinispan
+description: Builds Infinispan and Maven repository with all required artifacts
+
+runs:
+  using: composite
+  steps:
+    - name: Setup Java
+      uses: ./.github/actions/setup-java
+
+    - name: Build Infinispan
+      shell: bash
+      run: |
+        # Ensure this plugin is built first to avoid warnings in the build
+        ./mvnw install -Pdistribution -s maven-settings.xml
+        # By using "dependency:resolve", it will download all dependencies used in later stages for running the tests
+        ./mvnw install dependency:resolve -V -e -DskipTests -DexcludeGroupIds=org.infinispan -Dsilent=true -s maven-settings.xml
+
+    - name: Compress Infinispan Maven artifacts
+      shell: bash
+      run: |
+        tar -C ~/ --use-compress-program zstd -cf m2-infinispan.tzts \
+        --exclude '*.tar.gz' \
```

Comment:
shouldn't `zip` be excluded too?

Reply:
I don't think we produce any `*.tar.gz`, so I have just replaced this with `*.zip`